tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "typesafeconfig",
   "compilerOptions": {
     "module": "commonjs",
+    "moduleResolution": "Bundler",
```

Comment:
Added to allow TS to continue to pick up the correct types.

https://github.com/typescript-eslint/typescript-eslint/issues/7284

Contains a warning when building, but appears to still produce the same output.

Reply:
Does this issue persist into v8 of `@typescript-eslint/utils`?

Reply:
It is fixed! Removed the patch in https://github.com/shian15810/eslint-plugin-typescript-enum/pull/338/commits/34496c58dd7c75bf6243333122da57784961b0fc.